css/css-logical/logical-longhand-properties.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8" />
+<title>Test for handling of logical and physical properties</title>
+<link rel="author" title="Cameron McCormack" href="mailto:cam@mcc.id.au">
+<link rel="author" title="Oriol Brufau" href="mailto:obrufau@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-logical-1/#box">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<style id="sheet"></style>
+
+<!-- specify size for outer <div> to avoid unconstrained-size warnings
+     when writing-mode of the inner test <div> is vertical-* -->
+<div style="width:100px; height: 100px;">
+  <div id="test" class="test"></div>
+</div>
+
+<script>
+"use strict";
+const sheet = document.getElementById("sheet");
+const testElement = document.getElementById("test");
+
+// Logical properties with their descriptors.
+const logicalProperties = {
+  "border-*-color": {
+    type: "color",
+    prerequisites: {"color": "black"},
```

Comment:
What are these prerequistes?

Reply:
In this case I think it's because `border-*-color` defaults to `currentColor`, and the initial `color` value is UA-defined. So setting `color: black` you ensure that `border-*-color` won't be UA-defined. But here all tests that check `border-*-color` assign a value beforehand, so I can remove it.

In other cases it's needed in order to obtain a meaningful computed value, e.g. you can set `border-*-width` but if `border-*-style` is `none` then the former will compute to 0.